spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+RSpec.describe 'User post index page:', type: :feature do
+  before(:each) do
```

Comment:
- [ ] Good that you have all your views test passing :+1: But please ensure all your application tests are passing to ensure your app works properly. :100: Currently you have some of your tests failing so please kindly fix them :pray: 
  
  ![image](https://github.com/chepkok3/Blog_App/assets/92303891/dae86734-bcf5-4d3d-833b-326acd7d9eb5)

Reply:
@Owusu-Desmond Okay, let me fix them.

Reply:
@Owusu-Desmond Done